astropy/visualization/units.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 
-__all__ = ["quantity_support"]
+__all__ = ["quantity_support", "unit_support"]  # ✅ أضفنا unit_support هنا
```

Comment:
We only allow comments in english, so most contributors can understand them
```suggestion
__all__ = [
    "quantity_support",
    "unit_support",
]
```